plugins/modules/cloud/scaleway/scaleway_database_backup.py

```diff
@@ -0,0 +1,346 @@
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_database_backup
+short_description: Scaleway database backups management module
+author: Guillaume Rodriguez (@guillaume_ro_fr)
+description:
+    - This module manages database backups on Scaleway account
+      U(https://developer.scaleway.com)
+extends_documentation_fragment:
+    - community.general.scaleway
+options:
+  state:
+    description:
+        - Indicate desired state of the database backup.
+    type: str
+    default: present
+    choices:
+        - present
+        - absent
+        - exported
+        - restored
+
+  region:
+    description:
+        - Scaleway region to use (for example fr-par).
+    type: str
+    required: true
+    choices:
+        - fr-par
+        - nl-ams
+
+  id:
+    description:
+        - UUID used to identify the database backup.
+        - Required for C(absent), C(exported) and C(restored) states
+    type: str
+    required: false
+
+  name:
+    description:
+        - Name used to identify the database backup.
+        - Required for C(present) state
+    type: str
+    required: false
+
+  database_name:
+    description:
+        - Name used to identify the database.
+        - Required for C(present) and C(restored) states
+    type: str
+    required: false
+
+  instance_id:
+    description:
+        - UUID of the instance associated to the database backup
+        - Required for C(present) and C(restored) states
+    type: str
+    required: false
+
+  expires_at:
+    description:
+        - Expiration datetime of the database backup (ISO 8601 format)
+    type: str
+    required: false
+
+  wait:
+    description:
+        - Wait for the instance to reach its desired state before returning.
+    type: bool
+    default: 'no'
+
+  wait_timeout:
+    description:
+        - Time to wait for the backup to reach the expected state
+    type: int
+    required: false
+    default: 300
+
+  wait_sleep_time:
+    description:
+        - Time to wait before every attempt to check the state of the backup
+    type: int
+    required: false
+    default: 3
+'''
+
+EXAMPLES = '''
+  - name: Create a backup
+    scaleway_database_backup:
+        name: 'my_backup'
+        state: present
+        region: 'fr-par'
+        database_name: 'my-database'
+        instance_id: '50968a80-2909-4e5c-b1af-a2e19860dddb'
+
+  - name: Export a backup
+    scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: exported
+        region: 'fr-par'
+
+  - name: Restore backup
+    scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: restored
+        region: 'fr-par'
+        database_name: 'my-new-database'
+        instance_id: '50968a80-2909-4e5c-b1af-a2e19860dddb'
+'''
+
+RETURN = '''
+data:
+    description: This is only present when C(state=present)
```

Comment:
```suggestion
data:
    description: Backup metadata.
```
imo, it sounds like all data from a backup will be returned:) maybe `backup_info` or `backup_metadata` or something. And we have the info from `description` in `returned` subsection.